metricshub-agent/src/main/java/org/sentrysoftware/metricshub/agent/service/task/MonitoringTask.java

```diff
@@ -117,6 +117,12 @@ public void run() {
 			// Initialize the OpenTelemetry observers and LogEmitter after the discovery
 			// as at this time we should have what we want to observe
 			initOtelSdk(telemetryManager, resourceConfig);
+
+			// Initialize metric observers
```

Comment:
This requires an additional comment to explain why we are doing this here, please.